core/src/main/scala/org/apache/spark/status/LiveEntity.scala

```diff
@@ -625,10 +626,19 @@ private class SchedulerPool(name: String) extends LiveEntity {
 
 }
 
-private object LiveEntityHelpers {
+private[spark] object LiveEntityHelpers {
 
   private val stringInterner = Interners.newWeakInterner[String]()
 
+  private def accuValuetoString(value: Any): String = value match {
+    case list: java.util.List[_] =>
```

Comment:
Can we add a JIRA ID and why we only abbreviate this case alone?

Reply:
Yes. Good advice.